### PR TITLE
Eeitt 1079 secinclif

### PR DESCRIPTION
**Pull request by ralreiroe**
supporting includeIfs for section - wip

### CODE REVIEW

**Comment by pawelpanasewicz** [comment]
LGTM

File: app/uk/gov/hmrc/gform/core/parsers/ValueParser.scala [hunk 1]
```diff
@@ -21,7 +21,7 @@ import cats.instances.list._
 import cats.syntax.traverse._
 import parseback._
```
Comment (ralreiroe) [inline app/uk/gov/hmrc/gform/core/parsers/ValueParser.scala]: test

Comment (ralreiroe) [inline app/uk/gov/hmrc/gform/core/parsers/ValueParser.scala]: done

**Comment by ralreiroe** [comment]
merging after checking with Tenoch

**Pull request merged**